package.json

```diff
@@ -2,67 +2,63 @@
   "name": "dashjs",
   "version": "3.2.2",
   "description": "A reference client implementation for the playback of MPEG DASH via Javascript and compliant browsers.",
-  "main": "build/es5/index.js",
-  "types": "build/typings/index.d.ts",
+  "main": "dist/dash.all.debug.js",
```

Comment:
imo "main" should never be the build version but the entry point in the source code
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#main

candidates in our case are
* index.js
* index_mediaplayerOnly.js

Reply:
This avoids adding all source files in ouput module package.
This enables to link to main bundle file from integration project, thus (thanks to source maps) you will have dashjs files in sources panel from debug window under a "dashjs" node

Reply:
ok, thanks. With this use case in mind i think we should keep it this way